BitmarkRegistry/Controllers/Main/Properties/RegisterPropertyViewController.swift

```diff
@@ -130,25 +130,32 @@ extension RegisterPropertyViewController: UIImagePickerControllerDelegate, UINav
 // MARK: - UIDocumentPickerDelegate
 extension RegisterPropertyViewController: UIDocumentPickerDelegate {
   func documentPicker(_ controller: UIDocumentPickerViewController, didPickDocumentAt url: URL) {
-    let didStartAccessing = url.startAccessingSecurityScopedResource()
-    defer {
-      if didStartAccessing {
-        url.stopAccessingSecurityScopedResource()
+    activityIndicator.startAnimating()
+    disabledScreen.isHidden = false
+    UIApplication.shared.beginIgnoringInteractionEvents()
```

Comment:
It's deprecated? (https://developer.apple.com/documentation/uikit/uiapplication/1623047-beginignoringinteractionevents)

Reply:
Thanks, I will update soon.